src/main/java/com/develhope/spring/vehicles/service/VehicleService.java

```diff
@@ -26,6 +30,15 @@ public void deleteVehicleById(long vehicleId) {
         vehicleRepository.deleteById(vehicleId);
     }
 
+    public String findMostOrderedVehicleModel() {
```

Comment:
Non dovresti ritornare una String in questo metodo, secondo me è più consono ritornare una lista di tutti i veicoli con quel particolare modello

Reply:
io sto cercando specificamente solo il modello, indipendentemente dai veicoli e dal loro stato attuale, mi rendo conto che è tutto sbagliato, questa logica deve lavorare solo con la tabella order, le cose sono due: o si fa una custom queery e credo sia più immediato, o si crea un OrderServiceFilter così come abbiamo fatto per vehicle